proto/umee/incentive/v1/incentive.proto

```diff
@@ -10,7 +10,8 @@ option go_package = "github.com/umee-network/umee/v4/x/incentive";
 message Params {
   option (gogoproto.equal) = true;
 
-  // max_unbondings defines the maximum amount of concurrent unbondings an address can have.
+  // max_unbondings defines the maximum amount of concurrent unbondings an address can have
+  // on each unbonding tier of each bonded uToken denom.
```

Comment:
This change will improve both UX and implementation requirements, but results in a lower default value.